### PR TITLE
Add a type to the EOL property

### PR DESCRIPTION
**Pull request by MarkGibbons**
# Description

Describe what this change achieves

## Issues Resolved

Cookstyle warning about a missing type

## Check List

- [ ] All tests pass. See TESTING.md for details.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable.

### CODE REVIEW

**Comment by kitchen-porter** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please include a CHANGELOG entry.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ## 2.9.2 - *2020-12-06*
 
 - resolved cookstyle error: libraries/filter_helper.rb:59:11 convention: `Style/CommentAnnotation`
+- Add a type to the EOL property of the filter_lines resource
```
Comment (xorima) [inline CHANGELOG.md]: Hey @MarkGibbons 

This needs to go under the `## Unreleased` heading :) 

Thanks

**Review by xorima — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (MarkGibbons) [reply CHANGELOG.md]: Should be fixed now.  thanks

**Review by xorima — APPROVED** [review]

**Comment by kitchen-porter** [comment]
Released as: [2.9.3](https://github.com/sous-chefs/line/releases/tag/2.9.3)

**Pull request merged**